rally-domain/src/main/groovy/yakworks/rally/orgs/repo/ContactSourceRepo.groovy

```diff
@@ -73,4 +89,9 @@ class ContactSourceRepo extends LongIdGormRepo<ContactSource> {
         cs.persist()
         return cs
     }
+
+    boolean exists(String sourceId) {
+        if( !contactSourceExistsQuery ) contactSourceExistsQuery = KeyExistsQuery.of(getEntityClass()).keyName('sourceId')
```

Comment:
what is the unique key? what are we adding to the DB? I thought it was a combo not just sourceId

Reply:
its just sourceId in db @basejump